### PR TITLE
feat: add home page

### PR DESCRIPTION
**Pull request by Aditi0505**
# Description
- Added Home page 
- Added Images to display the screens

# Screenshots
Screenshot (74)

Screenshot (75)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* home layout */
+
+.outer-wrapper {
+  flex-direction: column;
+  gap: var(--sm-size);
+  padding: var(--sm-size);
+}
+.screen {
+  margin-top: 100px;
```
Comment (Naveenchandar) [inline styles.css:9]: Any reason to use px instead of em or rem ?

**Review by Naveenchandar — APPROVED** [review]
Looks good.

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: styles.css (hunk 1 above)
Comment (Aditi0505) [reply styles.css:9]: Oh I missed it, I will change it to rem.

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

**Pull request merged**